backend/GameManager.py

```diff
@@ -17,37 +19,44 @@ def get_board(self, row=-1, col=-1):
         return self.board.check_position(row, col)
 
     def get_turn(self):
-        if self.turn == Turn.WHITE:
-            return 'white'
-        else:
-            return 'black'
-    
+        return self.turn
+
+    def get_turn_as_string(self):
+        return self.turn.name.lower()
+
     def placement_complete(self):
         return self.pieces.all_pieces_placed()
 
-    def piece_placement(self, row, column):
+    def place_piece(self, row, column):
         if (row, column) in self.current_moves:
-            if self.board.check_position(row, column) != "empty":
-                print(self.board.check_position(row, column).lower())
+            if self.board.check_position(row, column) != CellType.EMPTY:
+                print(self.board.check_position(row, column))
                 return "GameManagerError -- position not empty"
-            
-            flag = 0
+
+            isPiecePlaced = 0
```

Comment:
This variable shouldn't be camelCase - rename is_piece_placed

Reply:
Updated in this commit 9c90828effbc08e240fd54975240748b60c7011d